manifest.outlook.xml

```diff
@@ -0,0 +1,97 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<OfficeApp
+  xmlns="http://schemas.microsoft.com/office/appforoffice/1.1"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xmlns:bt="http://schemas.microsoft.com/office/officeappbasictypes/1.0"
+  xsi:type="MailApp">
+  <Id>ac457896-2dd5-4ed3-ba7d-4319ae4929aa</Id>
+  <Version>1.0.0.0</Version>
+  <ProviderName>Contoso0</ProviderName>
+  <DefaultLocale>en-US</DefaultLocale>
+  <DisplayName DefaultValue="Office-Addin-NodeJS-SSO"/>
+  <Description DefaultValue="An add-in that shows how to use SSO, and to fallback to interactive login when SSO is not available."/>
+  <IconUrl DefaultValue="https://localhost:{PORT}/assets/icon-32.png"/>
+  <HighResolutionIconUrl DefaultValue="https://localhost:{PORT}/assets/icon-80.png"/>
+  <SupportUrl DefaultValue="https://www.contoso.com/help"/>
+	<Hosts>
+		<Host Name="Mailbox"/>
```

Comment:
Please check for tabs versus spaces. The indentation looks off. I think we typically have used spaces only for better formatting.